dali/operators/python_function/CMakeLists.txt

```diff
@@ -14,12 +14,13 @@
 
 set(python_function_lib "python_function_plugin")
 set(SOURCES python_function.cc dltensor_function.cc util/copy_with_stride.cc)
+
 add_library(${python_function_lib} SHARED ${SOURCES})
-target_link_libraries(${python_function_lib} ${dali_lib})
-target_include_directories(${python_function_lib}
-        PRIVATE ${PYBIND11_INCLUDE_DIR}  # from project CMakeLists.txt
-        PRIVATE ${pybind11_INCLUDE_DIR}  # from pybind11Config
-        PRIVATE ${PYTHON_INCLUDE_DIRS})
+target_link_libraries(${python_function_lib} PUBLIC ${dali_lib})
+target_include_directories(${python_function_lib} PRIVATE
+        ${PYBIND11_INCLUDE_DIR}  # from project CMakeLists.txt
+        ${pybind11_INCLUDE_DIR}  # from pybind11Config
```

Comment:
Why it cannot be private any longer?

Reply:
It is private, you don't need to repeat the keyword

Reply:
OK